doc/api/packages.md

```diff
@@ -67,12 +68,13 @@ field, or string input without the flag `--input-type`. This behavior is to
 preserve backward compatibility. However, now that Node.js supports both
 CommonJS and ES modules, it is best to be explicit whenever possible. Node.js
 will treat the following as CommonJS when passed to `node` as the initial input,
-or when referenced by `import` statements within ES module code:
+or when referenced by `import` statements, `import()` expressions, or
+`require()` expressions:
```

Comment:
What happens when you `require` a `.js` file in a `"type": "module"` scope? Does it error the way it does for `.mjs` files, or does it try to load the `.js` file as CommonJS in spite of the `"type"` field?

Reply:
It throws an `ERR_REQUIRE_ESM` error, same as with `.mjs` files.

```sh
mkdir repro
cd repro
touch file.js
node -e 'require("./file.js")'
echo '{"type":"module"}' > package.json
node -e 'require("./file.js")' # ERR_REQUIRE_ESM
```

Reply:
Excellent, I was hoping that’s what happened. 😌